lib/api/core/plugins/pluginContext.js

```diff
@@ -56,10 +61,12 @@ class PluginContext {
     this.constructors = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Group all shorthand properties at the beginning of this object declaration [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3499)